frontend/src/pages/profile-settings/components/subscription/components/subscription-management-dialog/subscription-management-dialog.tsx

```diff
@@ -0,0 +1,78 @@
+import { useCallback } from 'react';
+
+import { Button, Icon } from '#libs/components/components.js';
+import { AppRoute, IconColor } from '#libs/enums/enums.js';
+import { useAppDispatch } from '#libs/hooks/hooks.js';
+import { actions as appActions } from '#slices/app/app.js';
+
+import { PLAN_BENEFITS } from '../../libs/constants.js';
+import styles from './styles.module.scss';
+
+const dialogStyles = {
+  subscriptionManagement: styles['subscription-management'],
+  heading: styles['heading'],
+  benefits: styles['benefits'],
+  benefit: styles['benefit'],
+  content: styles['content'],
+  sectionTitle: styles['section-title'],
+  subscriptionName: styles['subscription-name'],
+  metaWrapper: styles['meta-wrapper'],
+  meta: styles['meta'],
+};
```

Comment:
I think it's overhead for us

Reply:
yeah, I agree. The reason is that the section-title class repeats a few times and eslint throws an error. So I decided to add all classes to the object to look consistent. Do I need to get rid of all classes but section-title?